app/src/main/java/com/github/h3lp3rs/h3lp/signin/GoogleSignInAdapter.kt

```diff
@@ -97,4 +98,17 @@ object GoogleSignInAdapter: SignInInterface<AuthResult> {
     override fun getUid(): String? {
         return auth.currentUser?.uid
     }
+
+    /**
+     *
+     */
+    fun getCreationDate(): String{
+        val timeStamp = auth.currentUser?.metadata?.creationTimestamp
+        return if(timeStamp==null){
+            ""
+        }else{
+            Date(timeStamp).toString()
+        }
```

Comment:
Using the let/run kotlin nullable operators makes it a little prettier but I'm not sure if it makes it less readable so this is only a suggestion
```suggestion
        return timeStamp?.let { Date(it).toString() } ?: run { "" }
```

Reply:
I will use it seems a more "Kotlin" way to do it